src/librustc_mir/borrow_check/mod.rs

```diff
@@ -2193,7 +2190,7 @@ impl<'cx, 'gcx, 'tcx> MirBorrowckCtxt<'cx, 'gcx, 'tcx> {
                                     // `*const` raw pointers are not mutable
                                     hir::MutImmutable => Err(place),
                                     // `*mut` raw pointers are always mutable, regardless of
-                                    // context. The users have to check by themselves.
+                                    // location. The users have to check by themselves.
```

Comment:
Should still be "context" (being used as a common English word)

Reply:
Thanks, I reverted wording.